### PR TITLE
Fix drone git@next plugin Gitea version display when building tag

### LINKED ISSUE #4359: 1.5.0-rc1 does not use 1.5.0-rc1 as version
- Gitea version (or commit ref): 1.5.0-rc1

## Description

Gitea shows as version in footer: "280ebcb" but it should be "1.5.0-rc1".
I'm using docker with `gitea/gitea:1.5.0-rc1`.

## Screenshots
screenshot-2018-7-4 gitea jonas franz software

### PR DESCRIPTION
**Pull request by lafriks**
Currently Drone CI when using git@next plugin to build tagged version gitea shows sha1 instead of actual version in footer.

Fixes #4359

See comments in issue for more details

### CODE REVIEW

**Review by jonasfranz — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,19 @@ GOFMT ?= gofmt -s
...
+ifneq ($(DRONE_TAG),)
+	VERSION ?= $(subst v,,$(DRONE_TAG))
+	GITEA_VERSION := $(VERSION)
+else
+	ifneq ($(DRONE_BRANCH),)
+		VERSION ?= $(subst release/v,,$(DRONE_BRANCH))
+	else
+		VERSION ?= master
+	endif
+  GITEA_VERSION := $(shell git describe --tags --always | sed 's/-/+/' | sed 's/^v//')
```
[the remark this answers is not in the document]
Comment (lafriks) [reply Makefile]: done

**Review by jonasfranz — APPROVED** [review]

**Comment by lunny** [comment]
conflict

**Comment by lafriks** [comment]
@lunny conflict resolved

**Comment by lafriks** [comment]
Drone failure is not related to PR

**Review by axifive — APPROVED** [review]

**Review by Bwko — APPROVED** [review]

**Pull request merged**